### PR TITLE
Create fixture with all workflows statuses

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Adds a new fixture that starts multiple workflows of each execution Status:

image

## Why?

Helps at developing Web UI by quickly populating sample data

## Checklist

1. Closes 

2. How was this tested:

Ran the fixture and verified on Web UI

3. Any docs updates needed?

The fixture contains a readme. No other docs updates needed

### CODE REVIEW

File: temporal-fixtures/rainbow-statuses/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This fixuture starts few sets workflows `(NumberOfSets)`- each set containing a workflow of each Status (ie. Open, Terminated, ..)
+
+Used in:
+
+-
```
Comment (feedmeapples) [inline temporal-fixtures/rainbow-statuses/README.md:5]:
```suggestion
- Web UI testing
```

File: temporal-fixtures/rainbow-statuses/starter/main.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	// "fmt"
```
Comment (feedmeapples) [inline temporal-fixtures/rainbow-statuses/starter/main.go:7]:
```suggestion
```

**Comment by Sushisource** [comment]
Is samples really the best place for this? I think we should prefer to keep samples constrained to... actual samples :smile:

**Comment by feedmeapples** [comment]
> Is samples really the best place for this? I think we should prefer to keep samples constrained to... actual samples

cc @sw-yx @stevekinney 
what do you think about moving fixtures to Web-v2 repo? https://github.com/temporalio/web-v2

**Review by sw-yx — APPROVED** [review]
re: fixture location, as discussed in slack

fixture looks great!!

File: temporal-fixtures/rainbow-statuses/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+This fixuture starts few sets workflows `(NumberOfSets)`- each set containing a workflow of each Status (ie. Open, Terminated, ..)
```
Comment (feedmeapples) [inline temporal-fixtures/rainbow-statuses/README.md:1]:
```suggestion
This fixuture starts few sets of workflows `(NumberOfSets)`- each set containing a workflow of each Status (ie. Open, Terminated, ..)
```

File: temporal-fixtures/rainbow-statuses/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+This fixuture starts few sets of workflows `(NumberOfSets)`- each set containing a workflow of each Status (ie. Open, Terminated, ..)
```
Comment (feedmeapples) [inline temporal-fixtures/rainbow-statuses/README.md:1]:
```suggestion
This fixuture starts few sets of workflows (`NumberOfSets`)- each set containing a workflow of each Status (ie. Open, Terminated, ..)
```

**Pull request merged**